apps/google-analytics-4/frontend/src/hooks/useKeyService.tsx

```diff
@@ -1,21 +1,31 @@
 import { useCallback, useState, useEffect } from 'react';
 import { AppExtensionSDK } from '@contentful/app-sdk';
 import { useSDK } from '@contentful/react-apps-toolkit';
-import { AppInstallationParameters, ServiceAccountKey, ServiceAccountKeyId } from 'types';
+import {
```

Comment:
Just beware that Lillian's changes also have a change to this hook. I'm pretty sure you guys won't conflict just take a look at her pr/discuss with her in person because whoever's pr gets in first will create a merge conflict here.